react/src/lib/Avatar/index.js

```diff
@@ -163,7 +163,7 @@ class Avatar extends React.Component {
       <div
         className={
           'md-avatar' +
-          `${(onClick && ` md-avatar--clickable`) || ''}` +
+          `${((onClick && type!=='bot') && ` md-avatar--clickable`) || ''}` +
```

Comment:
If the style isn't fixable then let's remove the extra parens here and add spaces around the '!=='

Reply:
@bfbiggs fixed